### PR TITLE
Application API has been changed to use app_control

### PR DESCRIPTION
**Pull request by WonyoungChoi**
Application API has been changed to use app_control_h instead of service_h.

https://review.tizen.org/gerrit/#/c/34955/

BUG=TC-2411

### CODE REVIEW

File: src/alarm/alarm_manager.cc [hunk 1]
```diff
@@ -199,40 +199,40 @@ int AlarmManager::GetAllAlarms(
...
-  if (service_destroy(service))
-    std::cerr << "Failed to call service_destroy()" << std::endl;
+void AlarmManager::DestroyAppControl(app_control_h app_control) const {
+  if (app_control_destroy(app_control))
+    std::cerr << "Failed to call app_control_destroy()" << std::endl;
 }
 
-int AlarmManager::StoreAlarmInService(service_h service,
+int AlarmManager::StoreAlarmInAppControl(app_control_h app_control,
                                       AlarmInfo* alarm) const {
```
Comment (deqing) [inline src/alarm/alarm_manager.cc]: Pls align with above

File: src/alarm/alarm_manager.cc [hunk 2]
```diff
@@ -199,40 +199,40 @@ int AlarmManager::GetAllAlarms(
...
     return -1;
 
-  return service_add_extra_data(service, kAlarmInfoKey,
+  return app_control_add_extra_data(app_control, kAlarmInfoKey,
       alarm->Serialize().c_str());
 }
 
-int AlarmManager::RestoreAlarmFromService(service_h service,
+int AlarmManager::RestoreAlarmFromAppControl(app_control_h app_control,
                                           AlarmInfo* alarm) const {
```
Comment (deqing) [inline src/alarm/alarm_manager.cc]: ditto

File: src/alarm/alarm_manager.cc [hunk 3]
```diff
@@ -35,20 +35,20 @@ int AlarmManager::ScheduleAlarm(const std::string& app_id,
...
+  int ret = StoreAlarmInAppControl(app_control, alarm);
   if (ret) {
-    DestroyService(service);
+    DestroyAppControl(app_control);
     std::cerr << "Failed to store the alarm information." << std::endl;
     return ret;
   }
 
-  ret = service_add_extra_data(service, kHostAppIdKey, host_app_id_.c_str());
+  ret = app_control_add_extra_data(app_control, kHostAppIdKey, host_app_id_.c_str());
```
Comment (deqing) [inline src/alarm/alarm_manager.cc]: Please split to two lines, it's longer than 80 chars

File: src/alarm/alarm_manager.cc [hunk 4]
```diff
@@ -247,20 +247,20 @@ int AlarmManager::RestoreAlarmFromService(service_h service,
...
+  DestroyAppControl(app_control);
 
   return passed;
 }
 
-bool AlarmManager::CheckOwnership(service_h service) const {
+bool AlarmManager::CheckOwnership(app_control_h app_control) const {
   char* buffer = NULL;
-  if (!service || service_get_extra_data(service, kHostAppIdKey, &buffer))
+  if (!app_control || app_control_get_extra_data(app_control, kHostAppIdKey, &buffer))
```
Comment (deqing) [inline src/alarm/alarm_manager.cc]: >80 chars, pls split

File: src/alarm/alarm_manager.h [hunk 5]
```diff
@@ -27,12 +27,12 @@ class AlarmManager {
...
 
  private:
-  service_h CreateAppLaunchService(const std::string& app_id) const;
-  void DestroyService(service_h service) const;
-  int StoreAlarmInService(service_h service, AlarmInfo* alarm) const;
-  int RestoreAlarmFromService(service_h service, AlarmInfo* alarm) const;
+  app_control_h CreateAppLaunchAppControl(const std::string& app_id) const;
+  void DestroyAppControl(app_control_h app_control) const;
+  int StoreAlarmInAppControl(app_control_h app_control, AlarmInfo* alarm) const;
+  int RestoreAlarmFromAppControl(app_control_h app_control, AlarmInfo* alarm) const;
```
Comment (deqing) [inline src/alarm/alarm_manager.h]: ditto, please split

**Comment by halton** [comment]
@WonyoungChoi please fix the trybot failure.

**Comment by WonyoungChoi** [comment]
The capi-appfw-application package should be applied to Tizen:Common repository first. 
SR is pending. 
https://build.tizen.org/search?utf8=%E2%9C%93&search_text=20150304.022845&commit=Submit&project=0&project=1&package=0&package=1&name=0&name=1&title=0&description=0&attribute=&issue_tracker=bnc+%28Novell+Bugzilla%29&issue

**Comment by halton** [comment]
@WonyoungChoi are you still working on this PR?

**Comment by WonyoungChoi** [comment]
This PR can't be merged because the SR for the fixed capi-appfw-application package was rejected to merge to Tizen:Common and Tizen:IVI. I'll close this PR for now.

**Pull request closed without merging**